api/api.go

```diff
@@ -238,6 +238,7 @@ func NewServerRoutes(s *Server) *http.ServeMux {
 	m.Handle("/history/", s.middleware(lh.LogHandler))
 
 	rgh := NewRegistryHandlers(s.qriNode)
+	m.Handle("/registry/list", s.middleware(rgh.RegistryListHandler))
```

Comment:
So glad to see this got merged! I do have one small nit here. I think this should be `/registry/datasets`, because the registry can also list users (more abstractly, the registry can `list` more than one type of thing)

Reply:
Got it, will make the change!